data/templates/post-windows-install.ps1

```diff
@@ -0,0 +1,30 @@
+<% if( undefined !== networkDevices ) { %>

```

Comment:
Ahh, yes, this need to be fix. Since networkDevices is optional, if networkDevices is not existing, this statement will throw error, but following is OK:
```
<%  if (typeof networkDevices !== 'undefined') { %>
```